src/operator/bufferToggle.ts

```diff
@@ -44,18 +46,18 @@ import {errorObject} from '../util/errorObject';
  * @owner Observable
  */
```

Comment:
We need to update docs in line 22 to say "an Observable or Promise returned by...".
And update docs in line 40 too.